x-pack/test/functional/services/infra_source_configuration_flyout.ts

```diff
@@ -81,6 +82,25 @@ export function InfraSourceConfigurationFlyoutProvider({
         await this.removeLogColumn(0);
       }
     },
+    async moveLogColumn(sourceIndex: number, destinationIndex: number) {
+      const logColumnPanel = (await this.getLogColumnPanels())[sourceIndex];
+      const moveLogColumnHandle = await testSubjects.findDescendant(
+        'moveLogColumnHandle',
+        logColumnPanel
+      );
+      await moveLogColumnHandle.focus();
+      const movementDifference = destinationIndex - sourceIndex;
+      await moveLogColumnHandle.pressKeys(Key.SPACE);
```

Comment:
Assuming we access the keys via the `browser` as suggested above, this could be

```suggestion
      await moveLogColumnHandle.pressKeys(browser.keys.SPACE);
```